src/sdk/namespace_fs.js

```diff
@@ -1536,6 +1546,31 @@ class NamespaceFS {
         }
     }
 
+    // handle xattr of content dir when moving from disabled to enabled mode.
+    // in the disabled version of content dir, the xattr is on the directory itself. so need to move it seperatly from the obejct
+    // in case of enabled mode we need to move the xattr to the new object
+    // both for suspended and enabled mode we need to clear the user xattr from the directory
+    async _handle_latest_disabled_dir_content_xattr(fs_context, key, versioned_path, latest_ver_path) {
```

Comment:
make sure to add jsdoc and not just comments, it helps a lot to the linter with finding issues